README.md

```diff
@@ -68,17 +68,17 @@ locales:
 
 ### Asset Precompilation
 
-Any .i18njs file can be added to your application's asset precompile list just as you might do with any other type of asset. However, it can be inefficient to include the translations for every language in your Javascript bundle, especially if you have lots of translations. To avoid this, i18n-js-assets supports splitting translations up into individual files, one per locale, for precompilation. Try something like this in your config/application.rb:
+Any `.i18njs` file can be added to your application's asset precompile list just as you might do with any other type of asset. However, it can be inefficient to include the translations for every language in your Javascript bundle, especially if you have lots of translations. To avoid this, i18n-js-assets supports splitting translations up into individual files, one per locale, for precompilation. Try something like this in your `config/application.rb`:
 
 ```ruby
 config.assets.localized.add(
-  'i18n/translations.js',
+  'i18n/translations.js.i18njs',
```

Comment:
Huh that's curious... I wonder if that behavior changed in Rails 5 😢  So hard to keep up with that stuff.